packages/wdio-spec-reporter/src/index.ts

```diff
@@ -27,12 +27,17 @@ export default class SpecReporter extends WDIOReporter {
         failed: '✖'
     }
 
+    private _sauceLabsSharableLinks:boolean = true
+
     constructor (options: SpecReporterOptions) {
         /**
          * make spec reporter to write to output stream by default
          */
         super(Object.assign({ stdout: true }, options))
         this._symbols = { ...this._symbols, ...this.options.symbols || {} }
+        this._sauceLabsSharableLinks = <boolean>('sauceLabsSharableLinks' in options
+            ? options.sauceLabsSharableLinks
+            : this._sauceLabsSharableLinks)
```

Comment:
I don't think casting is needed here, both should be booleans
```suggestion
        this._sauceLabsSharableLinks = 'sauceLabsSharableLinks' in options
            ? options.sauceLabsSharableLinks
            : this._sauceLabsSharableLinks
```

Reply:
👍

Reply:
I needed to cast them otherwise I need to accept `boolean` and `undefined` , what do you want

Reply:
How about this then:

```suggestion
        this._sauceLabsSharableLinks = 'sauceLabsSharableLinks' in options
            ? options.sauceLabsSharableLinks as boolean
            : this._sauceLabsSharableLinks
```

Reply:
Yes, this will work to, will fix it, thanks for the suggestion